components/public-api/gitpod/experimental/v1/user.proto

```diff
@@ -128,3 +130,13 @@ message GitToken {
     // value is the actual token value for the token
     string value = 7;
 }
+
+message BlockUserRequest {
+    // the ID of a User to block
+    string user_id = 1;
+
+    // reason is the reason for blocking the user
+    string reason = 2;
```

Comment:
We're not storing this today, but we can at least log it in our handler for now, until we start storing it.